invokeai/app/invocations/latent.py

```diff
@@ -73,8 +74,9 @@ class NoiseOutput(BaseInvocationOutput):
 
 
 def get_scheduler(scheduler_name:str, model: StableDiffusionGeneratorPipeline)->Scheduler:
-    scheduler_class = scheduler_map.get(scheduler_name,'ddim')
-    scheduler = scheduler_class.from_config(model.scheduler.config)
+    scheduler_class, scheduler_extra_config = scheduler_map.get(scheduler_name,'ddim')
```

Comment:
Fix it like:
```python
    if scheduler_name not in scheduler_map:
        scheduler_name = 'ddim'
    scheduler_class, extra_cfg = scheduler_map[scheduler_name]
```
As second argument is value to return if key not found, not alternative key.
So current code on unknown scheduler will return string 'ddim' instead of scheduler tuple.